lib/usd/ui/layerEditor/layerTreeModel.cpp

```diff
@@ -21,8 +21,10 @@
 #include "saveLayersDialog.h"
 #include "stringResources.h"
 #include "warningDialogs.h"
+#include <mayaUsd/utils/utilSerialization.h>
 
 #include <mayaUsd/utils/util.h>
+#include <mayaUsd/base/tokens.h>
```

Comment:
I have a feeling this might fail the clang-format check because it checks for grouping of includes. There should be blank line after "warningDialogs.h" and the three "<mayaUsd..." should be grouped together (no blank lines).